### PR TITLE
ADEN-7819 Run AE3 on FV pages

### PR DESCRIPTION
**Pull request by fraszczakszymon**

### CODE REVIEW

File: extensions/wikia/AdEngine3/src/slots.js [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+  return isSupportedPageType &&
+    window.wgIsContentNamespace &&
+    context.get('wiki.opts.adsInContent') &&
+    !context.get('wiki.targeting.wikiIsCorporate');
+}
+
+export default {
+  getContext() {
+    return {
+      TOP_LEADERBOARD: {
```
Comment (dmnsobczak) [inline extensions/wikia/AdEngine3/src/slots.js:30]: Don't we wanna switch slot names to lowercase?

**Review by dmnsobczak — APPROVED** [review]

File: extensions/wikia/AdEngine3/AdEngine3.setup.php [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<?php
+
+$wgExtensionCredits['other'][] = [
+	'name' => 'AdEngine',
+	'author' => 'Wikia',
+	'description' => 'AdEngine',
+	'version' => '1.0',
```
Comment (nandy-andy) [inline extensions/wikia/AdEngine3/AdEngine3.setup.php]: `3.0` ;)

File: extensions/wikia/AdEngine3/src/slots.js (hunk 1 above)
Comment (fraszczakszymon) [reply extensions/wikia/AdEngine3/src/slots.js:30]: Discussed offline. Not at this moment :)

File: extensions/wikia/AdEngine3/AdEngine3.setup.php (hunk 2 above)
Comment (fraszczakszymon) [reply extensions/wikia/AdEngine3/AdEngine3.setup.php]: Let it be :D

**Pull request merged**